### PR TITLE
Adx 961 verify that localization works with new validation framework

### PR DESCRIPTION
**Pull request by A-Souhei**
## Description

Added translations (FR, PT) for validations,
edited resource.config to reflect frictionless-components

No changes ! Below the comparison table

image

## Checklist

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request).
You may not need to check all boxes.

- [ ] The Jira ticket for this issue has been updated to "Ready to Review" or equivalent.
- [x] I have developed these changes in discussion with the appropriate project manager.
- [ ] My code follows the general Fjelltopp documentation (see Confluence).
- [ ] I have made corresponding changes to the Fjelltopp documentation (see Confluence).
- [ ] I have rebased this branch with master.
- [ ] New dependency changes have been committed.
- [ ] I have added automated tests that prove my fix is effective or that my feature works.
- [ ] New and existing tests pass locally with my changes.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.

### CODE REVIEW

File: ckanext/validation/i18n/fr/LC_MESSAGES/ckanext-validation.po [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+"POT-Creation-Date: 2023-07-13 12:23+0000\n"
+"PO-Revision-Date: 2023-07-13 12:23+0000\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: fr\n"
+"Language-Team: fr <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n > 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.7.0\n"
```
Comment (tomeksabala) [inline ckanext/validation/i18n/fr/LC_MESSAGES/ckanext-validation.po]: Please update the metadata to Fjelltopp, Toavina, info@fjelltopp.org

**Review by tomeksabala — DISMISSED** [review]

File: ckanext/validation/i18n/fr/LC_MESSAGES/ckanext-validation.po [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ckanext-validation 2.0.0\n"
+"Report-Msgid-Bugs-To: info@fjelltopp.org\n"
+"POT-Creation-Date: 2023-07-13 12:23+0000\n"
+"PO-Revision-Date: 2023-07-13 12:23+0000\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```
Comment (ChasNelson1990) [inline ckanext/validation/i18n/fr/LC_MESSAGES/ckanext-validation.po]: @A-Souhei, can you put you here?

File: ckanext/validation/i18n/pt_PT/LC_MESSAGES/ckanext-validation.po [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ckanext-validation 2.0.0\n"
+"Report-Msgid-Bugs-To: info@fjelltopp.org\n"
+"POT-Creation-Date: 2023-07-13 12:23+0000\n"
+"PO-Revision-Date: 2023-07-13 12:25+0000\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```
Comment (ChasNelson1990) [inline ckanext/validation/i18n/pt_PT/LC_MESSAGES/ckanext-validation.po]: And here?

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
Just a wee change or two please

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**